### ISSUE TITLE
Doesn't do anything on very large json

### ISSUE THREAD

**Issue by Kiina** [richie5um/vscode-sort-json#9]
So I have a json file with about half a million lines and about 16 million characters. When I execute the command nothing happend but I don't get an error. I tested a short snippet from my json in another document and there it works. So it's not a problem with the json itself.

Here is the example which works, the real json is like this but with half a million lines

```
[
    {
        "lastVisitTime": "2017-07-22T18:09:41.706Z",
        "id": "328649",
        "title": "Sort JSON objects - Visual Studio Marketplace",
        "visitCount": 1,
        "url": "https://marketplace.visualstudio.com/items?itemName=richie5um2.vscode-sort-json",
        "typedCount": 0
    },
    {
        "lastVisitTime": "2017-07-22T18:09:19.794Z",
        "id": "328648",
        "title": "richie5um/vscode-sort-json",
        "visitCount": 1,
        "url": "https://github.com/richie5um/vscode-sort-json",
        "typedCount": 0
    }
]
```

**Comment by richie5um** [comment]
Hi. I suspect it might be a memory problem with a file of that size.

I'm assuming that the file can be successfully opened by VS Code?

Can you send me the file so I can test it? Or does it contain sensitive data?

Thanks
Rich

**Comment by Kiina** [comment]
Yes, VS Code has no problem with the file itself (which is amazing because Atom just crashed if I try to save). Unfortunately I can't send you the file because of sensitive data.

**Comment by Kiina** [comment]
Update:

Okay it has nothing todo with the json size it seems. If I open the .json in VS Code it doesn't work. If I just copy the complete json and paste it in a new document (without saving etc) I can sort without a problem. But then I can't save the file after that
